native/ex_tokenizers/src/models.rs

```diff
@@ -362,23 +363,34 @@ pub fn models_wordlevel_from_file(
 #[derive(NifTaggedEnum)]
 pub enum UnigramOption {
     UnkId(usize),
+    ByteFallback(bool),
 }
 
 #[rustler::nif]
 pub fn models_unigram_init(
     vocab: Vec<(String, f64)>,
     options: Vec<UnigramOption>,
 ) -> Result<ExTokenizersModel, ExTokenizersError> {
-    let unk_id = if !options.is_empty() {
-        match options[0] {
-            UnigramOption::UnkId(unk_id) => Some(unk_id),
-        }
-    } else {
-        None
+    let unk_id = match options
+        .iter()
+        .find(|opt| matches!(opt, UnigramOption::UnkId(_)))
+        .unwrap()
+    {
+        UnigramOption::UnkId(unk_id) => Some(*unk_id),
+        _ => None,
+    };
+
+    let byte_fallback = match options
+        .iter()
+        .find(|opt| matches!(opt, UnigramOption::ByteFallback(_)))
```

Comment:
`iter . find` on each opt is O(n**2) complexity.
Probably it's better to find another solution.

Reply:
With two options it's fine, but we could also do the same as here:

https://github.com/elixir-nx/tokenizers/blob/170ceac858063feb9ef1092e27917478489c5896/native/ex_tokenizers/src/models.rs#L116-L125

Reply:
Yeah I mean we're talking about an incredibly negligible bit of performance, but yeah mutating an opts struct is probably more performant :).